dbcrossbarlib/src/drivers/postgres/csv_to_binary/mod.rs

```diff
@@ -158,27 +158,27 @@ fn array_to_binary(
     let mut buffer = vec![];
     wtr.write_value(&mut buffer, |wtr| {
         // The number of dimensions in our array.
-        wtr.write_i32::<NE>(dimension_count)?;
+        WriteBytesExt::write_i32::<NE>(wtr, dimension_count)?;
```

Comment:
this seems slightly less clean but I haven't investigated the context

Reply:
Yeah, this is caused by conflicting `write_i32` methods defined by two different libraries, one new, so I have to invoke this the long way now.